example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```

Comment:
As the comment in this line states, do not check this into version control. Also contains personal data about your computer / folder structure.

Reply:
Sounds good to me!